CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to Safe Core Protocol
+
+Even if you don't write code, there are many ways you can help improve the project.
+
+Here are some important resources:
+
+  * [Safe docs](https://docs.safe.global) tells you about Safe
+  * [Safe{Core} Protocol Specification](https://github.com/safe-global/safe-core-protocol-specs) tells you about the protocol specifications.
+
+## Submitting issues
+
+If you are submitting a feature request, please:
+
+  * Explain in detail the purpose and how it should work.
+  * Examples of how it should be implemented along with alternatives are welcome.
+
+If you are submitting a bug report, please:
+  * Include the steps to reproduce the bug.
+  * Include the expected behavior.
+  * Possible solutions to fix the bug are welcome.
+
+## Submitting changes
+
+Please send a [GitHub Pull Request to safe-core-protocol repository](https://github.com/safe-global/safe-core-protocol) with a clear description of the proposed changes. Each pull request should be associated with an issue and should be made against the `main` branch.
```

Comment:
Do we expect external contributors to open an issue if it does not exist so far? Or are the branching conventions primarily for the core maintainers of the repo (so the devs that have write access to this repo).

Reply:
Yes, I think there should be issue for every PR and even external contributors should open one if does not exist.